README.md

```diff
@@ -14,6 +14,24 @@ Please see the [MLPerf Inference benchmark paper](https://arxiv.org/abs/1911.025
 }
 ```
 
+## MLPerf Inference master
```

Comment:
r1.0 has a README without the master section ... you don't want to include this file.

Reply:
@guschmue  can you help fix this? or can you let me merge for now and I will update the readme in a separate PR? Somehow my local git account is unhappy and keeps telling me "remote: Permission to mlcommons/inference.git denied to christ1ne." It is very strange as I can do PR via the web interface here.

Reply:
Or we can first merge this and then @guschmue removes this part in a separate PR?

Reply:
actually, let me just merge it ... the readme is not going to kill us.